lib/storyboards/widgets/guess_image_story.dart

```diff
@@ -14,7 +14,7 @@ class GuessImageStory extends FullScreenStory {
             child: Container(
               color: Colors.white,
               child: GuessImage(
-                imageName: "assets/hospital.jpg",
+                imageName: "assets/topic/asb/hospital.jpg",
```

Comment:
i meant use any existing image from topic/asb. If you want to add temporary image for testing, you can create a temp folder inside assets

Reply:
done